### ISSUE TITLE
How can code obfuscation be avoided in development mode?

### ISSUE THREAD

**Issue by asasas234** [wxt-dev/wxt#328]
I noticed that even in development mode, the code is being obfuscated. I went through the documentation but couldn't find any information on how to set it so that the code doesn't get obfuscated. It can be quite challenging to debug the code when it's obfuscated.

When setting up the project skeleton, I opted for TypeScript and Vue.

**Comment by aklinker1** [comment]
Vite is doing the bundling and minifying, and you can customize it's behavior by using the [`vite` option](https://wxt.dev/api/wxt/interfaces/InlineConfig.html#vite) in your `wxt.config.ts`.

```ts
import { defineConfig } from 'wxt';

export default defineConfig({
  vite: ({ command }) => ({
    build: {
      minify: command === "build",
    },
  }),
});
```

See Vite's [`build.minify` option](https://vitejs.dev/config/build-options.html#build-minify) for more details.

I'll include this fix in WXT and make a new release. You can add the above config yourself, or wait a few hours for my to add it and release a new version.

**Comment by asasas234** [comment]
@aklinker1 If a new version is released, please notify here.

**Comment by aklinker1** [comment]
Released in [`v0.13.4`](https://github.com/wxt-dev/wxt/releases/tag/v0.13.4)
